pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -165,7 +165,7 @@ public class MultiTopicsConsumerImpl<T> extends ConsumerBase<T> {
             return;
         }
 
-        checkArgument(topicNamesValid(conf.getTopicNames()), "Topics is invalid.");
+        checkArgument(topicNamesValid(conf.getTopicNames()), "The param [topics] is invalid.");
```

Comment:
I suggest using "Subscription topics include duplicate items or invalid names." instead.

Reply:
There are two points here:

1. Duplicate topics
2. The topic name is invalid

and the `param [topics]` only applies to Java client.

That's why I put forward the suggestion.

Reply:
Sure, fixed.